src/main.rs

```diff
@@ -126,6 +126,7 @@ fn run(mut config: Config, options: &cli::Options) -> Result<(), Box<Error>> {
     // pseudoterminal. A file descriptor for the master side is retained for
     // reading/writing to the shell.
     let mut pty = tty::new(&config, options, &display.size(), window_id);
+    display.window().show();
```

Comment:
This should probably use `display.present()` instead so that it's decoupled from `window`.